Formula/volk.rb

```diff
@@ -4,8 +4,8 @@ class Volk < Formula
   desc "Vector Optimized Library of Kernels"
   homepage "https://www.libvolk.org/"
   url "https://github.com/gnuradio/volk.git",
-      tag:      "v2.5.0",
-      revision: "237a6fc9242ea8c48d2bbd417a6ea14feaf7314a"
+      tag:      "v2.5.1",
+      revision: "ce4e4a00fc953f3451bfcdd4310b03785d8b4055"
```

Comment:
The only Git submodule is `cpu_features`, so we might be able to revert back to source tarball